src/components/code-block.tsx

```diff
@@ -69,6 +69,10 @@ function DocsCodeBlock({
 		metastring ? `${className} ${metastring}` : className,
 	);
 
+	if (_title) {
+		meta.title = _title;
```

Comment:
This was broken for quite some time. Not only is this used by the triple backquote style blocks, but this ends up getting used directly when using `CodeBlock` from `@themes/CodeBlock` and it wasn't respecting the `title` prop, when it should